docs/mkdocs.yml

```diff
@@ -7,7 +7,26 @@ nav:
   - At Sea Operation: atseaoperation.md
   - ROS 2 Interface: ros2.md
   - Simulation: simulation.md
-  - Tutorials: tutorials.md
+  - Tutorials:
```

Comment:
perhaps reorder tutorials to go at the bottom of the nav sidebar. The stuff after tutorials appears as if they are also under the`Tutorials` heading in the nav bar